spade-core/src/main/scala/com/salesforce/mce/spade/cli/GenerateCommand.scala

```diff
@@ -0,0 +1,30 @@
+package com.salesforce.mce.spade.cli
+
+import io.circe.syntax._
+
+import com.salesforce.mce.spade.orchard.WorkflowRequest
+import com.salesforce.mce.spade.SpadeWorkflowGroup
+
+class GenerateCommand(opt: CliOptions, workflowGroup: SpadeWorkflowGroup) {
+
+  def run(): Int = {
+    val requests = for {
+      (workflowKey, workflow) <- workflowGroup.workflows
```

Comment:
seems that `workflowKey` gets lost, is it intended?

Reply:
@anniezhang77 yeah I saw that, I could `_` it since we are not using it, thought `workflowKey` is a good indication of what it is also